student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, *args, **kwargs)
```

Comment:
- [ ] There is no need to process hashes or arrays on this class, you only need to use the same attribute that you are using for the `Person` class plus the `classroom` attribute, so it should be more like this:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)
```

Reply:
if i do it that way, which I did (parent_permission = true) => everything works fine except for linters
if I put it as parent_permission: true as linters suggest for booleans I get error for arguments when testing

Reply:
(given 3, expected 1..2)

Reply:
You can leave the parent_permission like this `parent_permission: true`, but please don't use arrays or hashes, they are not necessary 😄

Reply:
ok I will not use them
This is what I had before using args and kwargs

`require_relative './person'
class Student < Person
  attr_accessor :classroom

  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission)
    @classroom = classroom
  end

  def play_hooky
    "¯\(ツ)/¯"
  end
end`

and I believe it's what is demanded but when I do

`user = ('Class A', 17, 'username')` I get an error of arguments which will only disappear if I discard parent_permission
should I push even if the creating a user test doesn't work ?

Reply:
Please show me a screenshot of the error 👾

Reply:
thank you very much for bearing with me. I really appreciate it.
this is my code for students.rb

![image](https://user-images.githubusercontent.com/83329154/146066658-5d629965-5e44-4b5f-ad2b-9f3d160cd261.png)

this is the error message : 

![image](https://user-images.githubusercontent.com/83329154/146066789-bc9fe19c-4e7f-4157-8e63-8ced072c31d0.png)

this is also my person.rb in case that would cause something :

![image](https://user-images.githubusercontent.com/83329154/146066916-4816b3c5-330a-493e-8d0a-b204d1122db4.png)

the error shows when I try to create an instance of Student